### PR TITLE
Added Javadocs and Made Changes to SendMessageUseCase

### PR DESCRIPTION
**Pull request by whyismynamerudy**
Made some changes to SendMessageUseCase to accommodate some restrictions and limitations due to Clean Architecture when initializing the class in ChatViewPresenter. Still need to add tests for chat functionality, will continue to add tests to the branch after this.

### CODE REVIEW

**Review by skoooch — APPROVED** [review]
Great job adding javadocs!

File: src/main/java/UseCases/ChatUseCases/SendMessageUseCase.java [hunk 1]
```diff
@@ -1,20 +1,59 @@
...
+        User curr_user = user_graph.getUser(user.getUser());
+
+        Set<User> users = new HashSet<>();
+        users.add(user_graph.getUserByString(user1));
+        users.add(user_graph.getUserByString(user2));
+
+//        //user 1 is current user
+//        if (user1.equals(curr_user.getUsername().data)){
+//            Message new_message = new Message,
+//        }
```
Comment (jay-ead) [inline src/main/java/UseCases/ChatUseCases/SendMessageUseCase.java:49]: Not sure why this is commented out.

**Review by jay-ead — APPROVED** [review]
All the java docs look good, and the SendMessageUseCase seems to be efficient and clean. Fantastic work Rudy!

**Review by Ary4m3n — APPROVED** [review]
The JavaDocs look coherent, relevant and complete. I believe just writing the tests are left now. Good Work!

**Pull request merged**